oteljava/all/src/main/scala/org/typelevel/otel4s/oteljava/OtelJava.scala

```diff
@@ -25,6 +25,12 @@ import cats.syntax.all._
 import io.opentelemetry.api.{OpenTelemetry => JOpenTelemetry}
 import io.opentelemetry.api.GlobalOpenTelemetry
 import io.opentelemetry.sdk.{OpenTelemetrySdk => JOpenTelemetrySdk}
+import io.opentelemetry.sdk.autoconfigure.{
+  AutoConfiguredOpenTelemetrySdk => AutoConfigOtelSdk
+}
+import io.opentelemetry.sdk.autoconfigure.{
+  AutoConfiguredOpenTelemetrySdkBuilder => AutoConfigOtelSdkBuilder
```

Comment:
I thought these aliases appeared in Scaladoc, but proved to myself that's not the case.  I don't mind them as long as they don't add any public aliases.

Reply:
having the type of the function be two lines long was just too much for me